controllers/octavia_controller.go

```diff
@@ -599,6 +599,10 @@ func (r *OctaviaReconciler) reconcileNormal(ctx context.Context, instance *octav
 	// Amphora reconciliation
 	// ------------------------------------------------------------------------------------------------------------
 
+	if err = octavia.EnsureQuotas(ctx, instance, &r.Log, helper); err != nil {
+		return ctrl.Result{}, err
+	}
```

Comment:
A suggestion for a follow-up PR might be to add conditions to house and surface the error here and also on L617.  Maybe we could reuse the condition from L625 (`amphoraControllerReadyCondition(octaviav1.HealthManager)`)?  That way the user could see the problem in `oc get octavia` without having to search through `oc logs <Octavia controller manager pod>`.